mne_bids/inspect.py

```diff
@@ -122,9 +170,18 @@ def _save_bads_dialog_box(*, bads, bids_path, verbose):
     from mne.viz.utils import figure_nobar
 
     title = 'Save changes?'
-    message = (f'You have modified the bad channel selection of\n'
-               f'{bids_path.basename}.\n\nWould you like to save these '
-               f'changes to the\nBIDS dataset?')
+    message = 'You have modified '
+    if bads is not None and annotations is None:
+        message += 'the bad channel selection '
+    elif bads is None and annotations is not None:
+        message += 'the bad segments selection '
+    else:
+        message += 'the bad channel and\nsegments selection '
```

Comment:
when you modify both the text is overlapping in the dialog box

Reply:
Interesting, not for me. Could you share a screenshot? Which backend are you using?

Reply:
Ping @agramfort, I would appreciate a screenshot if you have a minute :)

Reply:
<img width="975" alt="Screenshot 2020-11-28 at 20 14 15" src="https://user-images.githubusercontent.com/161052/100524195-7152e900-31b6-11eb-8eff-ad61a1db5ca0.png">

Reply:
Thanks!

Reply:
Which backend is this?

Reply:
@agramfort I've pushed a change, could you please try again?